gammapy/scripts/tests/test_image_pipe.py

```diff
@@ -65,8 +65,8 @@ def test_image_pipe(tmpdir):
                          obs_table=data_store.obs_table, exclusion_mask=exclusion_mask)
     mosaic.make_images(make_background_image=True, for_integral_flux=True, radius=10.)
     # TODO: this fails on some machines, for unknown reasons.
-    #assert_equal(mosaic.maps['counts'].data.sum(), 2334.0)
-    #assert_almost_equal(mosaic.maps['bkg'].data.sum(), 1987.2088283446637)
-    #assert_almost_equal(mosaic.maps['exposure'].data.sum(), 5.9292301086257267e17)
-    #assert_almost_equal(mosaic.maps['significance'].lookup(center), 33.699068269123039)
-    #assert_almost_equal(mosaic.maps['excess'].data.sum(), 346.79117165533626)
+    assert_equal(mosaic.maps['counts'].data.sum(), 2334.0)
```

Comment:
The other build fail on travis-ci is unrelated.
This should be ready to be merged once you make the `assert` change I commented on above.